packages/shopify-app-remix/src/server/authenticate/login/__tests__/login.test.ts

```diff
@@ -117,7 +118,7 @@ describe('login helper', () => {
 
         // THEN
         const expectedPath = testCaseConfig.redirectToInstall
-          ? `https://${TEST_SHOP}/admin/oauth/install?client_id=${config.apiKey}`
+          ? `https://admin.shopify.com/store/${TEST_SHOP_NAME}/oauth/install?client_id=${config.apiKey}`
```

Comment:
I'll add another refactor PR to use the new utility from [this PR](https://github.com/Shopify/shopify-api-js/pull/1093) once that's ready